extension/src/test/fixtures/plotsDiff/index.ts

```diff
@@ -936,10 +1004,60 @@ const getIndFromComparisonMultiImgPath = (path: string) => {
   return Number((pathIndMatches as string[])[1])
 }
 
+const boundingBoxColors = ['#ff3838', '#ff9d97', '#ff701f']
+
+export const collectPlotClasses = ({
+  plotClasses,
+  classLabels,
+  boundingBoxes,
+  id,
+  path
+}: {
+  plotClasses: ComparisonPlotClasses
+  classLabels: Set<string>
+  boundingBoxes: { label: string; box: BoundingBox }[]
+  id: string
+  path: string
+}) => {
+  const classAcc: {
+    [label: string]: {
+      label: string
+      boxes: {
+        h: number
+        w: number
+        x: number
+        y: number
+      }[]
```

Comment:
Why isn't this reusing [this](https://github.com/iterative/vscode-dvc/pull/5250/files#diff-97ae15b18dd87d9190444f391683d3bc0d68c9187d1af5af01870e7631ea323dR247) type or the `BoundingBox ` one?

Reply:
Good catch!

Reply:
Can't you change `w`, `x`, `h` and `y` on the frontend as well so that we have a continuity throughout the extension?

Reply:
> Can't you change w, x, h and y on the frontend as well so that we have a continuity throughout the extension?

Good point, that would simplify things! I'll update the code so that instead of the backend performing the calculations, the calculations are done in `ComparisonTableBoundingBoxImg`.

Reply:
Done!